src/main/java/com/taskadapter/redmineapi/internal/RedmineJSONBuilder.java

```diff
@@ -33,180 +33,179 @@
 public class RedmineJSONBuilder {
 
     /**
```

Comment:
cannot review this part - too many changes in RedmineJSONBuilder. what happened to this class? why are hundreds of lines deleted and hundreds added? bad reformatting or moved methods?